packages/@aws-cdk/tsconfig.json

```diff
@@ -0,0 +1,100 @@
+{
+    "__comment__": "This file is necessary to make transitive Project References in TypeScript work",
+    "files": [],
+    "references": [
+        { "path": "alexa-ask" },
```

Comment:
Can we generate this file upon `install.sh`?

Reply:
Hmm, potentially. Better to do it in `build.sh` I think then, otherwise adding a package will break the build until you run `install.sh` which is going to be very unintuitive.

Reply:
Anything but manually requiring people to edit this